.github/workflows/ui-components.yml

```diff
@@ -0,0 +1,34 @@
+name: Node.js ui-components CI
+
+env:
+  context: ./packages/ui-components/
+
+on:
+  push:
+    branches: [main]
+    tags: "*"
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - run: yarn install --frozen-lockfile
```

Comment:
IMO this install should be done only for the ui-components project, to decrease the install time